cmd/selflow-daemon/selflow_daemon.go

```diff
@@ -30,7 +32,7 @@ func setupLogger() {
 func main() {
 	setupLogger()
 
-	configAsBytes, err := os.ReadFile("/etc/selflow/config.yaml")
+	configAsBytes, err := os.ReadFile("/home/anthony/Projets/selflow/internal/config/testdata/flow.yaml")
```

Comment:
😨

Reply:
![You saw nothing](https://media.tenor.com/pQEJeg8Rd0IAAAAC/the-simpsons-homer-simpson.gif)